launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -26,6 +26,11 @@
 #include "../../lib/CConfigHandler.h"
 #include "../../lib/Languages.h"
 
+inline double mbToBytes(double mb)
```

Comment:
- Use static for free functions
- Despite its name, "inline" is not about compiler optimization, but about linking multiple compilation units - no need for "inline" in local functions.
```suggestion
static double mbToBytes(double mb)
```